Userland/Libraries/LibCore/System.cpp

```diff
@@ -298,20 +301,27 @@ ErrorOr<int> anon_create([[maybe_unused]] size_t size, [[maybe_unused]] int opti
         TRY(close(fd));
         return Error::from_errno(saved_errno);
     }
-#elif defined(__APPLE__)
+#elif defined(__APPLE__) || defined(AK_OS_WIN32)
     struct timespec time;
     clock_gettime(CLOCK_REALTIME, &time);
     auto name = String::formatted("/shm-{}{}", (unsigned long)time.tv_sec, (unsigned long)time.tv_nsec);
+#if defined(AK_OS_WIN32)
+    fd = ::open(String::formatted("/dev/shm{}", name).characters(), O_RDWR | O_CREAT | options, 0600);
```

Comment:
this.. is... bad. You are opening a file in `c:\dev\shm{}` on windows... how does it work? At least open a file in the user directory...

Reply:
This is part of msys2. Not windows

Reply:
oh.. I was thinking on mingw :)